app/src/androidTest/java/com/github/sdp/ratemyepfl/utils/MenuUtils.kt

```diff
@@ -40,7 +40,7 @@ fun openSearchView(@StringRes searchViewId: Int) {
 /**
```

Comment:
Why not put these methods in an `object`? (also applies for `RecyclerViewUtils`)

Reply:
At first, I passed the id in the constructor, and when I changed I did not payed attention to it. I'll do it.